test/e2e/test/context.go

```diff
@@ -101,6 +101,7 @@ type Context struct {
 	TestRun             string              `json:"test_run"`
 	AutoPortForwarding  bool                `json:"auto_port_forwarding"`
 	Local               bool                `json:"local"`
+	LogToFile           bool                `json:"log_to_file"`
```

Comment:
I don't think this is needed because none of the tests depend on it.

Reply:
Done